extra/teamcollision.txt

```diff
@@ -13,7 +13,7 @@
 			"CGameMovement::PlayerSolidMask"
 			{
 				"linux"		"16"
-				"windows"	"17"
+				"windows"	"15" // was 17 (pointing to CGameMovement::CalcRoll)
```

Comment:
I apologize, you've misunderstood my meaning, a gamedata file of project's source code is not a place for comments on changes, all of comments go into "conversation" tab on a github website. 😀

Basically, you need to remove your comment from the file completely, leaving changed offset value within double quotes as the only edit.

Reply:
Ah okay, I got it.